package.xml

```diff
@@ -8,6 +8,7 @@
 
   <!-- Build dependencies -->
   <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_python</buildtool_depend>
   <build_depend>rosidl_default_generators</build_depend>
```

Comment:
Could we also document somewhere in the main README the need for rosidl_default_generators?